MarkdownToLatex/MarkdownToLatex/MdToTex.cs

```diff
@@ -35,9 +35,11 @@ internal static void convertMathElement(Match match)
                     foreach(Match m in mc){
                         switch(m.Groups[1].Value){
                             case "result":
-                                double param;
+                                double param; 
+                                int precision;
                                 bool hasParam = double.TryParse(m.Groups[2].Value, NumberStyles.AllowLeadingSign | NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out param);
-                                LatexRenderer.WriteMathElement((calc as FuncCalculator).Calculate(hasParam ? param : 0));
+                                bool hasPrecision = int.TryParse(m.Groups[3].Value, out precision);
+                                LatexRenderer.WriteMathElement((calc as FuncCalculator).Calculate(hasParam ? param : 0, hasPrecision ? precision : 2));
```

Comment:
In diesem Fall kannst du eigentlich noch in `FuncCalculator.Calculate()` die `= 2` vom Parameter entfernen, da hier der Standardwert festgelegt wird.

Reply:
Achso stimmt